guestbook/comments/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from math import trunc
+
+# Create your models here.
+
+def calc_long_ago(created_at):
+    delta = timezone.now() - created_at
+    print(created_at, timezone.now(), delta.seconds)
+    if delta.days > 15:
+        return "a while ago"
+    else:
+        if delta.days > 1:
+            t = trunc(delta.days)
+            return "{:d} day{} ago".format(t, "" if t > 1 else "s")
+        else:
+            if delta.seconds > 3600:
+                t = trunc(delta.seconds/60/60)
+                return "{:d} hours ago".format(t, "" if t > 1 else "s")
+
+            elif delta.seconds > 300:
+                t = trunc(delta.seconds/60)
+                return "{:d} minute{} ago".format(t, "" if t > 1 else "s")
+            else:
+                return "just now"
+
+class Comment(models.Model):
+    author = models.ForeignKey(User)
+    title = models.CharField(max_length=200)
+    text = models.CharField(max_length=1024)
```

Comment:
Do you really want a char field or an arbitrary long text? In the latter case you have to use a `models.TextField`